pkg/util/blacklist.go

```diff
@@ -0,0 +1,22 @@
+package helpers
+
+import (
+	"errors"
+	"strings"
+)
+
+var blacklistedCommands = []string{
+	"FLUSHALL", "FLUSHDB", "DUMP", "ABORT", "AUTH", "CONFIG", "SAVE", "BGSAVE",
+	"BGREWRITEAOF", "RESTORE", "MULTI", "EXEC", "DISCARD", "QWATCH", "QUNWATCH",
+	"LATENCY", "CLIENT", "SLEEP", "PERSIST",
+}
+
+// IsBlacklistedCommand checks if a command is blacklisted
+func IsBlacklistedCommand(cmd string) error {
```

Comment:
Please rename this to `BlockListedCommand`

Reply:
ok